saleor/graphql/schema.graphql

```diff
@@ -5043,19 +5044,33 @@ enum WeightUnitsEnum {
 
 type Wishlist implements Node {
   id: ID!
-  createdAt: DateTime!
-  items(before: String, after: String, first: Int, last: Int, id: ID): WishlistItemCountableConnection!
+  token: UUID!
+  user: User
```

Comment:
Do we want to have wishlist without user?

Reply:
After discussion: we should remove the user field from wishlist type.